rotkehlchen/chain/evm/decoding/hop/structures.py

```diff
@@ -0,0 +1,26 @@
+from dataclasses import dataclass
+
+from rotkehlchen.types import ChecksumEvmAddress
+
+
+@dataclass
```

Comment:
can you also set the standard dataclass keys like we do with other dataclasses to determine if it's mutable/immutable if it has setters/getters etc.

And if it can just be immutable you can make it a named tuple.

Reply:
It's going to be immutable, changed it to NamedTuple.